components/raftstore-v2/tests/failpoints/test_pd_heartbeat.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 TiKV Project Authors. Licensed under Apache-2.0.
+
+use futures::executor::block_on;
+use pd_client::PdClient;
+use raftstore_v2::router::{StoreMsg, StoreTick};
+use tikv_util::config::ReadableDuration;
+
+use crate::cluster::{v2_default_config, Cluster};
+
+#[test]
+fn test_fake_store_heartbeat() {
+    fail::cfg("mock_collect_tick_interval", "return(0)").unwrap();
+
+    let cluster = Cluster::with_config_and_extra_setting(v2_default_config(), |config| {
+        config.pd_store_heartbeat_tick_interval = ReadableDuration::millis(10);
```

Comment:
How about setting these configs before running? So we do not need the new `with_config_and_extra_setting`.

Reply:
Yep, I struggled from it for a long time, but it can't be.

In previous work, the `Cluster::with_config` will automatically force reset all tick with ZERO.
![image](https://github.com/tikv/tikv/assets/18441614/fe8e4dd5-26dc-460b-8c5a-571a556ed6dd)

So, if we do not import the extra `with_config_and_extra_setting`, expected tick on `fake` store heartbeat won't be triggered.